src/lib/components/toggle/QToggle.svelte

```diff
@@ -1,31 +1,39 @@
 <script lang="ts">
   import type { QToggleProps } from "./props";
 
-  export let value: QToggleProps["value"],
-    icon: QToggleProps["icon"] = undefined,
-    label: QToggleProps["label"] = undefined,
-    leftLabel: QToggleProps["leftLabel"] = undefined,
-    disable: QToggleProps["disable"] = undefined,
-    userClasses: QToggleProps["userClasses"] = "";
-  export { userClasses as class };
+  let {
+    value = $bindable(),
+    icon = undefined,
+    label = undefined,
+    leftLabel = undefined,
+    disable = undefined,
+    ...props
+  }: QToggleProps = $props();
 
   function toggle() {
     if (disable !== true) {
```

Comment:
Then you can write `if (!disable) {`

Reply:
Also, why the if statement?

Reply:
It wasn't changed as you can see. The `if` is probably to only reassign when the condition is met, instead of running every time regardless.